pkg/pillar/types/zedmanagertypes.go

```diff
@@ -159,6 +161,9 @@ type AppInstanceStatus struct {
 	State          SwState
 	MissingNetwork bool // If some Network UUID not found
 	MissingMemory  bool // Waiting for memory
+
+	EffectiveActivate bool //set here effective activate after profile set
```

Comment:
"after profile check"? Or did you mean something different?

Reply:
Fixed